ui/src/kapacitor/components/TelegramConfig.js

```diff
@@ -50,49 +55,76 @@ const TelegramConfig = React.createClass({
       <div>
         <h4 className="text-center no-user-select">Telegram Alert</h4>
         <br/>
-        <p className="no-user-select">You can have alerts sent to Telegram by entering info below.</p>
+        <p className="no-user-select">
+          Send alert messages to a <a href="https://docs.influxdata.com/kapacitor/v1.2/guides/event-handler-setup/#telegram-bot" target="_blank">Telegram bot</a>.
+        </p>
         <form onSubmit={this.handleSaveAlert}>
           <div className="form-group col-xs-12">
-            <label htmlFor="url">Telegram URL</label>
-            <input className="form-control" id="url" type="text" ref={(r) => this.url = r} defaultValue={url || ''}></input>
-          </div>
-
-          <div className="form-group col-xs-12">
-            <label htmlFor="token">Token</label>
-            <input className="form-control" id="token" type="text" ref={(r) => this.token = r} defaultValue={token || ''}></input>
+            <label htmlFor="token">
+              Token
+              <QuestionMarkTooltip
+                tipID="token"
+                tipContent="<p>Need help finding your token? Check out <a href='https://docs.influxdata.com/kapacitor/latest/guides/event-handler-setup/#telegram-api-access-token' target='_blank'>these steps</a>.</p>"
+              />
+            </label>
+            <input
+              className="form-control"
+              id="token"
+              type="text"
+              placeholder="your-telegram-token"
+              ref={(r) => this.token = r}
+              defaultValue={token || ''}>
+            </input>
             <label className="form-helper">Note: a value of <code>true</code> indicates the Telegram token has been set</label>
           </div>
 
           <div className="form-group col-xs-12">
-            <label htmlFor="chat-id">Chat ID</label>
-            <input className="form-control" id="chat-id" type="text" ref={(r) => this.chatID = r} defaultValue={chatID || ''}></input>
+            <label htmlFor="chat-id">
+              Chat ID
+              <QuestionMarkTooltip
+                tipID="chat-id"
+                tipContent="<p>Need help finding your chat id? Check out <a href='https://docs.influxdata.com/kapacitor/latest/guides/event-handler-setup/#telegram-chat-id' target='_blank'>these steps</a>.</p>"
```

Comment:
What do you think about pulling this text into a `/copy` directory and importing them?  Or, some other strategy that doesn't include this long of a string in a prop?